generator/internal/api/model.go

```diff
@@ -226,6 +435,101 @@ type Message struct {
 	IsPageableResponse bool
 	// PageableItem is the field to be paginated over.
 	PageableItem *Field
+
+	// The API that this message belongs to.
+	API *API
+
+	// Elements is a map of all the elements in the message, keyed by their name.
+	// This field is only available after the message is initialized.
```

Comment:
I find "initialized" a bit confusing.  What do you think of "cross-referenced" ?  And then it would be "CrossReferenceVisitor" instead of "InitializationVisitor"?

Reply:
Changed the name and documentation